src/BulletCollision/BroadphaseCollision/btCollisionAlgorithm.h

```diff
@@ -64,11 +63,11 @@ class btCollisionAlgorithm
 	
 public:
 
-	btCollisionAlgorithm() {};
+	btCollisionAlgorithm() = default;
 
 	btCollisionAlgorithm(const btCollisionAlgorithmConstructionInfo& ci);
 
-	virtual ~btCollisionAlgorithm() {};
+	virtual ~btCollisionAlgorithm() = default;
```

Comment:
this [breaks the build](https://travis-ci.org/bulletphysics/bullet3/jobs/58096284#L365-L367):
~~~
/home/travis/build/bulletphysics/bullet3/src/BulletCollision/BroadphaseCollision/btCollisionAlgorithm.h:70:36: error: ‘virtual btCollisionAlgorithm::~btCollisionAlgorithm()’ declared virtual cannot be defaulted in the class body
make[2]: *** [src/BulletCollision/CMakeFiles/BulletCollision.dir/BroadphaseCollision/btCollisionAlgorithm.o] Error 1
make[2]: *** Waiting for unfinished jobs....
~~~